src/kindhandlers/ReplicaSet.handler.ts

```diff
@@ -19,9 +19,9 @@ const ReplicaSetHandler: ResourceKindHandler = {
     const k8sAppV1Api = kubeconfig.makeApiClient(k8s.AppsV1Api);
     return k8sAppV1Api.readNamespacedReplicaSet(resource.name, resource.namespace || 'default', 'true');
   },
-  async listResourcesInCluster(kubeconfig: k8s.KubeConfig) {
+  async listResourcesInCluster(kubeconfig: k8s.KubeConfig, { namespace }) {
     const k8sAppV1Api = kubeconfig.makeApiClient(k8s.AppsV1Api);
-    const response = await k8sAppV1Api.listReplicaSetForAllNamespaces();
+    const response = await k8sAppV1Api.listNamespacedReplicaSet(namespace);
```

Comment:
I'm wondering if we should have two different methods in the kind handlers, one for calling when we know the user has full access, and one for getting resources from specific namespace like this one.
I'm just not sure if there's a difference in performance or not